### PR TITLE
fix: loyalty prd changes

### PR DESCRIPTION
**Pull request by yuesth**
- added tooltip when unlock button is disabled
- added wording "add more ..$PARAS to be .. member" when first time to lock
- disabled 30 days duration when previous locked has duration 90 days
- added current member on main page
- added duration when unlock
- added "learn more about paras locked staking"

### CODE REVIEW

File: components/Modal/UnlockedStakeTokenModal.tsx [hunk 1]
```diff
@@ -25,12 +28,39 @@ const UnlockedStakeTokenModal = (props: UnlockedStakeModalProps) => {
...
+			return 'Bronze'
+		} else if (value >= SILVER && value < GOLD) {
+			return 'Silver'
+		} else if (value >= GOLD && value < PLATINUM) {
+			return 'Gold'
+		} else if (value >= PLATINUM) {
+			return 'Platinum'
+		}
+	}
+
```
Comment (ahnafalfariza) [inline components/Modal/UnlockedStakeTokenModal.tsx:51]: kayaknya currentMemberLevel bisa dibikin function helper aja sih menurut gua

File: components/Modal/UnlockedStakeTokenModal.tsx [hunk 2]
```diff
@@ -99,34 +135,84 @@ const UnlockedStakeTokenModal = (props: UnlockedStakeModalProps) => {
...
+								onClick={() =>
+									onClickDuration(process.env.NEXT_PUBLIC_APP_ENV === 'mainnet' ? 90 : 9)
+								}
+								className={`border rounded-lg p-1 px-2 text-white text-xs transition-all ${
+									(duration === 90 || duration === 9) && `bg-blueButton hover:bg-blue-600`
+								} ${
+									Number(inputValue) === maxToUnlock
+										? `border-gray-400 bg-gray-400 cursor-default`
+										: `border-blueButton cursor-pointer`
+								}`}
```
Comment (ahnafalfariza) [inline components/Modal/UnlockedStakeTokenModal.tsx:210]: kayaknya bisa langsung pakein clsx deh wkwkw, biar ga bingung bacanya
https://www.npmjs.com/package/clsx

File: components/MainPool.tsx [hunk 3]
```diff
@@ -496,13 +510,24 @@ const MainPool = ({ data, staked, stakedNFT, type, filterType = 'all', className
...
-				setLockedDuration(90)
+				const diffStartedEnded =
+					lockedBalanceDetails[0].ended_at - lockedBalanceDetails[0].started_at ===
+					THREE_MINUTES_IN_SECONDS
+				if (diffStartedEnded) {
+					setLockedDuration(3)
+				} else {
+					setLockedDuration(9)
+				}
 			}
```
Comment (ahnafalfariza) [inline components/MainPool.tsx:531]: ini kenapa ga langsung pake selisihnya dari smart contract aja gan?

File: components/Modal/UnlockedStakeTokenModal.tsx (hunk 1 above)
Comment (yuesth) [reply components/Modal/UnlockedStakeTokenModal.tsx:51]: woke fixing gan

File: components/Modal/UnlockedStakeTokenModal.tsx (hunk 2 above)
Comment (yuesth) [reply components/Modal/UnlockedStakeTokenModal.tsx:210]: woke gan

File: components/MainPool.tsx (hunk 3 above)
Comment (yuesth) [reply components/MainPool.tsx:518]: di smart contract nya cuma ada return started_at sama ended_at nya gan, jadinya nge compare nya dari situ

**Pull request merged**